parcel.go

```diff
@@ -24,9 +26,28 @@ type ParcelParams struct {
 	PhoneNumber      string
 	ExternalID       string
 	ToServicePointID int64
-	Weight           int64
 	OrderNumber      string
 	SenderID         int64
+
+	// Weight contains the weight in Kilograms. Supports up to 3 decimals
+	// (grams). For example: decimal.New(20, -3) specifies 20 grams.
+	Weight decimal.Decimal
```

Comment:
This incurs a breaking change in the API of this package. I don't like breaking changes so initially tried to make it backwards compatible but found that it was impossible to do that because previous behavior was unspecified.

The Weight field here was not passed to SendCloud. It was just ignored. There is no documentation on the field that specifies whether it is expected to have the weight in Kilograms, grams, pounds, whatever. So if we were to silently start sending this value to SendCloud it might cause issues with existing users of the package that use a different unit from whatever we choose to convert/send (e.g. they set it in grammes, we send it in Kilos).
Therefore I propose to explicitly break existing users of this package, forcing them to validate their code and explicitly pass a decimal.Decimal value in Kilograms.

Reply:
Optional:
As a convenience for users, I propose to add a version tag (v0.1.0?) on the current master branch, before merging this PR. This makes that users of this package may easily pin that version and aren't forced to upgrade. From there on a new version must be tagged to allow users to upgrade. After merging this PR the master branch should be tagged as v0.2.0.

Reply:
I agree that external dependencies is something to avoid. In this case I needed it because the Sendcloud API places a constraint on the number of decimal places that can be sent. Otherwise it returns this error:

> 2021/08/03 17:01:02 failed to create a new parcel at sendcloud: request api/v2/parcels resulted in error code 400: weight: "Ensure that there are no more than 3 decimal places."

Also, a float is not an accurate decimal type. It shouldn't be used for things like weight and money/value.